### DISCUSSION TITLE
How to display/hide/destroy tray icon dynamically?

### DISCUSSION THREAD

**Discussion by Cowjiang** [tauri-apps/tauri#10180] [Q&A] [answered]
I'm trying to add the ability to hide or show the tray icon by a switcher in UI. Seems I could use AppHandle.tray_handle().destroy() to remove it, but I don't know how to make it display again....

Do I need to do initialation like the first time again? Or any other better way?

**Comment by FabianLars** [comment, marked as answer]
> Do I need to do initialation like the first time again?

In v1, yes. In v2 there's a set_visibile function too.
